store/badger.go

```diff
@@ -132,6 +154,39 @@ func (b *badgerKV) CommitWriteBatch(batch WriteBatch) error {
 	return wb.batch.Flush()
 }
 
+func (b *badgerKV) gc(interval time.Duration) {
+	b.closeWg.Add(1)
+
+	go func() {
+		defer b.closeWg.Done()
+
+		ticker := time.NewTicker(interval)
+		defer ticker.Stop()
+
+		for {
+			select {
+			case <-b.closeCh:
+				return
+			case <-ticker.C:
+				for {
```

Comment:
why there is another for loop? can't we just make gc interval shorter?

Reply:
It's an optimization in the documentation.
https://github.com/dgraph-io/badger#garbage-collection